skipruntime-ts/core/native/src/Extern.sk

```diff
@@ -669,8 +669,8 @@ fun subscribeOfRuntime(
       reactiveAuth,
     )
   }) match {
-  | Success(id) -> SKJSON.CJString(id.toString())
-  | Failure(err) -> SKJSON.CJFloat(getErrorHdl(err))
+  | Success(id) -> id
+  | Failure(err) -> -getErrorHdl(err).toInt()
```

Comment:
Why is it negative now?

Reply:
Ok, but call sites do not seem to be modified in this PR?

Reply:
This is why it's a fix the call sites consider already the return value is a bigint, probably something introduced during a merge.